tests/Services/Search/ParametersHandlerTest.php

```diff
@@ -280,10 +383,18 @@ public function testHandlePaginationAndSorting(
             ->setMethods([])
             ->getMock();
 
+        $returnedUri = 'test.com';
+
         if ($isSearch) {
-            $requestMock->expects($this->once())->method('getUri')->willReturn('test.com/search');
+            $returnedUri .= '/search';
         }
 
+        if ($isFiltersSet) {
+            $returnedUri .= '?attrib';
```

Comment:
To make this more realistic, add some fake value to it. Only having the key is not really realistic.